node/clients/sellers.ts

```diff
@@ -19,7 +39,69 @@ export default class SellersClient extends JanusClient {
     })
   }
 
-  public async getSellers(): Promise<{ items: Seller[] }> {
-    return this.http.get(SELLERS_PATH)
+  public async getSellers(args?: GetSellersOpts): Promise<GetSellersResponse> {
+    const argsWithDefaults = {
+      ...INITIAL_LIST_OPTIONS,
+      ...args,
+    }
+
+    const result = await this.http.get<{
+      paging: {
+        from: number
+        to: number
+        total: number
+      }
+      items: Seller[]
+    }>(SELLERS_PATH, {
+      metric: 'sellers-get',
+      params: {
+        from: argsWithDefaults?.page ?? INITIAL_LIST_OPTIONS.page,
+        to: argsWithDefaults?.pageSize ?? INITIAL_LIST_OPTIONS.pageSize,
+      },
+    })
+
+    if (!result.items) {
+      return {
+        items: [],
+        pagination: {
+          page: 0,
+          pageSize: 0,
+          total: 0,
+        },
+      }
+    }
+
+    if (result.items.length > 1) {
+      result.items = this.sortSellers(result.items)
+    }
+
+    return {
+      items: result.items,
+      pagination: {
+        page: argsWithDefaults.page,
+        pageSize: argsWithDefaults.pageSize,
+        total: result.paging.total,
+      },
+    }
+  }
+
+  private sortSellers(sellers: Seller[]) {
+    return orderBy(
+      sellers,
+      [
+        (seller: Seller) => {
+          const name = seller.name ?? ''
+          const isNumber = !isNaN(name) && name !== ''
+
+          return isNumber ? 1 : 0
+        },
+        (seller: Seller) => {
+          const name = seller.name ?? ''
+
+          return isNaN(name) ? name : Number(name)
+        },
+      ],
+      ['asc', 'asc']
+    )
```

Comment:
can you add a comment to the code explaining a bit how this sort works?

Reply:
Done!